Remove DDF option from Feedback (Satellite) (#2552)
theforeman/foreman-documentation: guides/common/modules/proc_providing-feedback-on-red-hat-documentation.adoc

[preface]

[id="providing-feedback-on-red-hat-documentation_{context}"]
= Providing Feedback on Red Hat Documentation

We appreciate your input on our documentation.
Please let us know how we could make it better.

You can submit feedback by filing a ticket in Bugzilla:

. Navigate to the link:https://bugzilla.redhat.com/enter_bug.cgi?product=Red%20Hat%20Satellite[Bugzilla] website.

> . In the *Component* field, use `Documentation`.

Author: Sorry, Adam. I understand that "use" is generic, but "input" just sounds strange to me.

We appreciate your input on our documentation. Please let us know how we could make it better.

You can submit feedback through Bugzilla.

> To create a new ticket:

Reviewer: ```suggestion
```
I think line 8 is enough (if you end it with a colon).

. Go to the link:https://bugzilla.redhat.com/enter_bug.cgi?product=Red%20Hat%20Satellite[Bugzilla] website.

> . As the *Component*, use `Documentation`.

Reviewer 2: ```suggestion
. In the *Component*, use `Documentation`.
```

> . In the *Description* field, enter your suggestion for improvement. Include a link to the relevant part(s) of documentation.

Reviewer: ```suggestion
. In the *Description* field, enter your suggestion for improvement. Include a link to the relevant parts of documentation.
```
Alternatively, you could write "part or parts", but this ^ is what RHEL docs use.

Reviewer 2: Also, can we follow one sentence per line here?
